bindings/java/mongocrypt/build.gradle.kts

```diff
@@ -43,7 +43,7 @@ repositories {
 }
 
 group = "org.mongodb"
-version = "1.8.0-SNAPSHOT"
+version = "1.7.0-SNAPSHOT"
```

Comment:
Should this stay as 1.8 on the `master` branch and only switch to 1.7 when this is cherry-picked to the 1.7 branch?

Reply:
Ack